### DISCUSSION TITLE
Restore Purchase on new Mac

### DISCUSSION THREAD

**Discussion by Theauxm** [noah-nuebling/mac-mouse-fix#1114] [Questions]
Hello. I purchased MMF on an old work laptop. I recently received a new one. Do I have to buy MMF again on the new computer? Or am I able to get my activation key somewhere from my old Macbook?

**Comment by noah-nuebling** [comment]
Hi! The license key is saved in your iCloud so it should just work. If not, you should be able to find the license key in your emails. If neither of that works, let me know and I'll try to solve the issue.
